scripts/docker/fedora/33/Dockerfile

```diff
@@ -95,8 +97,20 @@ RUN useradd \
     --gid ${JENKINS_GROUPID} \
     --shell "/bin/bash" \
     jenkins
+
+# download and install gradle
+RUN cd /tmp && wget https://services.gradle.org/distributions/gradle-6.8.3-bin.zip && unzip gradle-6.8.3-bin.zip && rm gradle-6.8.3-bin.zip && mv gradle-6.8.3 /opt/gradle
+ENV PATH "${PATH}:/opt/gradle/bin"
+RUN echo "export JAVA_HOME=$(dirname $(dirname $(readlink $(readlink $(which javac)))))" >> /etc/profile.d/java.sh
+RUN echo "export PATH=$PATH:$JAVA_HOME/bin" >> /etc/profile.d/java.sh
+RUN echo "CLASSPATH=.:$JAVA_HOME/jre/lib:$JAVA_HOME/lib:$JAVA_HOME/lib/tools.jar" >> /etc/profile.d/java.sh
+RUN source /etc/profile.d/java.sh
```

Comment:
How much of this is truly needed? At least the `PATH` is set correctly by default afaik.

Reply:
@mpranj the initial problem was JAVA_HOME not being set. i will check whether i can omit updating the PATH

Reply:
Ok, the cleanup can also be done in a follow-up PR.

Reply:
It seems you are right and we don't need it after all... I'm not sure how i got the impression that we do, but it was late, that evening ;)
I've removed it from the fedora Dockerfiles for now.